### PR TITLE
Updated service point documentation & fixed taxes description

### PR DESCRIPTION
**Pull request by ChristianCasado**
JIRA TASKS:
https://auctane.atlassian.net/browse/ENGINE-5308
https://auctane.atlassian.net/browse/ENGINE-5312

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -5593,6 +5589,20 @@ components:
       description: >
         A [shipping carrier](https://www.shipengine.com/docs/carriers/setup/),
         such as `fedex`, `dhl_express`, `stamps_com`, etc.
+    lat:
```
Comment (agustinmartinpck) [inline openapi.yaml:5592]: Why are not using here `$ref: '#/components/schemas/lat'`?

File: openapi.yaml [hunk 2]
```diff
@@ -5593,6 +5589,20 @@ components:
...
+      format: double
+      type: number
+      example: 48.842608
+      description: >-
+        The latitude of the point. Represented as signed degrees. Required
+        if long is provided. http://www.geomidpoint.com/latlon.html
+    long:
```
Comment (agustinmartinpck) [inline openapi.yaml:5599]: Why are not using here `$ref: '#/components/schemas/long'`?

**Review by agustinmartinpck — APPROVED** [review]

File: openapi.yaml (hunk 1 above)
Comment (ChristianCasado) [reply openapi.yaml:5592]: this one is 'components/schemas/lat'  and where the other descriptions came from

File: openapi.yaml (hunk 2 above)
Comment (ChristianCasado) [reply openapi.yaml:5599]: this one is 'components/schemas/long' and where the other descriptions came from

**Review by afidalgomoran82 — APPROVED** [review]

**Pull request merged**